src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -110,7 +110,8 @@ protected AcquireTokenHandlerBase(RequestData requestData)
             brokerParameters[BrokerParameter.Resource] = requestData.Resource;
             brokerParameters[BrokerParameter.ClientId] = requestData.ClientKey.ClientId;
             brokerParameters[BrokerParameter.CorrelationId] = this.CallState.CorrelationId.ToString();
-            brokerParameters[BrokerParameter.ClientVersion] = AdalIdHelper.GetAdalVersion();
+            brokerParameters[BrokerParameter.ClientVersion] = AdalIdHelper.GetAssemblyFileVersion();
```

Comment:
>AdalIdHelper.GetAssemblyFileVersion(); [](start = 62, length = 38)

we should have a task to set this from the build system instead of us having to specify this manually... a task in the technical debt user story perhaps